ports/nrf/mpconfigport.h

```diff
@@ -312,11 +304,6 @@ extern const struct _mp_obj_module_t music_module;
     { MP_ROM_QSTR(MP_QSTR_help), MP_ROM_PTR(&mp_builtin_help_obj) }, \
     { MP_ROM_QSTR(MP_QSTR_open), MP_ROM_PTR(&mp_builtin_open_obj) }, \
 
-// extra constants
-#define MICROPY_PORT_CONSTANTS \
-    { MP_ROM_QSTR(MP_QSTR_board), MP_ROM_PTR(&board_module) }, \
-    { MP_ROM_QSTR(MP_QSTR_machine), MP_ROM_PTR(&mp_module_machine) }, \
```

Comment:
This is needed so the parser can optimise constants from the `machine` module.  Eg:
```python
print(machine.HARD_RESET)
```
That constant is looked up at parse time, not runtime.

Reply:
Okay, reverted that part.